src/obfl-specification.html

```diff
@@ -2925,15 +2962,81 @@ <h4 id="L2475">pre-translated</h4>
-    U+0020</li>
+  <li>BRAILLE PATTERN BLANK (U+2800) is not regarded as white space and should
+    be preserved. Proper white space characters must be used at word boundaries
+    in order to achieve correct line breaking.</li>
   <li>pre-translated text nodes are not influenced by the hyphenate attribute
     nor the global hyphenation policy</li>
   <li class="markup">style elements MUST NOT occur in pre-translated
   context</li>
 </ul>
 
+<!--
+<h4 id="L2475">bypass</h4>
+
+<p>Just like 'pre-translated', the value 'bypass' indicates that no braille
+transcription must be applied to the text. The difference with 'pre-translated'
+is that the text must not be processed at all.</p>
+-->
+
+<h4 id="L2479">grade:<em>&lt;number&gt;</em></h4>
+
+<p>The value 'grade:<em>&lt;number&gt;</em>', where <em>&lt;number&gt;</em> is a
+non-negative integer or decimal number, indicates that a braille code must be
+applied with the specific contraction grade identified by the number. Note that
+how contraction grades are numbered is region dependent.</p>
+
+<h4 id="L2483">6-dot</h4>
+
+<p>The value '6-dot' indicates that a braille code must be used that does not
+make use of dots 7 and 8.</p>
+
+<h4 id="L2487">8-dot</h4>
+
+<p>The value '6-dot' indicates that a braille code must be used that does makes
+use of dots 7 and 8.</p>
+
+<h4 id="L2491">Combinations</h4>
+
+<p>Combinations of above values can be made by concatenating them, with a '/'
+character as delimiter. Not all combinations are valid however, and the parts
+must appear in a specific order. The combination must exist of one or more of
+the following, in the given order:</p>
+
+<ul>
+  <li>'contracted' or 'uncontracted'</li>
+  <li>grade:<em>&lt;number&gt;</em></li>
+  <li>'6-dot' or '8-dot'</li>
+</ul>
+
+<p>Some examples:</p>
+
+<ul>
+  <li>grade:1.5</li>
+  <li>uncontracted/6-dot</li>
+  <li>grade:2/8-dot</li>
+</ul>
+
+<h3 id="L2500">xml:lang</h3>
+
+<p>The <code>xml:lang</code> attribute is used to specify the language in which
+the content is written. It does <strong>NOT</strong> imply association with a
+particular braille code (this is the role of the <a href="#L2462"><code
+>translate</code></a> attribute). A braille code may however have special rules
+for transcribing text written in another language than its "principal"
+language.</p>
+
+<p>The language specified by <code>xml:lang</code> applies to all elements in
+its content unless overridden with another instance of <code>xml:lang</code>.
+In particular, the empty value of <code>xml:lang</code> is used on an element B
+to override a specification of <code>xml:lang</code> on an enclosing element A,
+without specifying another language. Within B, it is considered that there is
+no language information available, just as if <code>xml:lang</code> had not
+been specified on B or any of its ancestors.</p>
+
+<p>The values of the attribute are language identifiers as defined by
+[<a class="nref" href="#ref-bcp47">BCP 47</a>].</p>
```

Comment:
Both links in [BCP-47] are obsolete. They point to non-existing web pages. See below

Reply:
Fixed.